File: C/Atividade18.c
#include <stdio.h>
#include <conio.h>
#include <stdlib.h>
#include<stdio.h>
#include<time.h> 
#include<string.h>
#include<locale.h>  

int main()
{
	float altura, peso, IMC, sexo;
	
	printf ("SENDO:\nMASCULINO=1\nFEMININO=2\n\nSEXO:");
    scanf ("%f", &sexo);
	
    printf ("\n\nAltura em metros:");
    scanf ("%f", &altura);
    
    printf ("\n\nPeso em kg:");
    scanf ("%f", &peso);

    IMC = peso/(altura*altura);

    printf ("\n\nO Indice de Massa Corporal (IMC) de uma pessoa com %.2f kg e %.2f metros de    altura e de %.2f", peso, altura, IMC);
   
   if(sexo==1)
   {   
    	if (IMC<20)
    	{
    		printf ("\n\nEsta pessoa esta abaixo do peso");
		}
		else 
		{
			if (IMC>25)
			{
				printf ("\n\nEsta pessoa esta acima do peso");
			}
			else
			{
				printf ("\n\nEsta pessoa esta no peso ideal");
			}
		}
	}
	
	if (sexo==2)
   {   
    	if (IMC<19)
    	{
    		printf ("\n\nEsta pessoa esta abaixo do peso");
		}
		else 
		{
			if (IMC>24)
			{
				printf ("\n\nEsta pessoa esta acima do peso");
			}
			else
			{
				printf ("\n\nEsta pessoa esta no peso ideal");
			}
		}
	}

    return 0;
}

File: Python/Atividade18.py
sexo = (input("Sexo: "))

if (sexo==("m") or sexo==("f")):

    peso = float(input("Peso: "))
    altura = float(input("Altura: "))
 
    IMC= peso/(altura*altura)

    print("O IMC dessa pessoa é de",IMC)

    if (sexo==("m")):
        if (IMC>20):
            if(IMC>25):
                print ("Essa pessoa não está no peso ideal")

            else:
                print ("Essa pessoa está no peso ideal")
        
        else:
            print ("Essa pessoa não está no peso ideal")

    else:
        if (IMC>19):
            if(IMC>24):
                print ("Essa pessoa não está no peso ideal")

            else:
                print ("Essa pessoa está no peso ideal")
        
        else:
            print ("Essa pessoa não está no peso ideal")


else:
    print ("Sexo inválido!")
    
